53.md

```diff
@@ -84,6 +84,79 @@ Event `kind:1311` is live chat's channel message. Clients MUST include the `a` t
 
 Hosts may choose to pin one or more live chat messages by updating the `pinned` tags in the live event kind `30311`.
 
+### Stream Raids
+
+Event `kind:1312` is a live stream raid (redirect viewers to another stream).
+
+Clients MUST include two `a` tags, one marked as `from` and the other marked as `to` which redirects viewers to a new live stream.
+The `content` MAY contain a raid message.
+
+Clients SHOULD automatically redirect viewers from one stream to another. 
+This is a similar function used on Twitch also called raids.
+
+```jsonc
+{
+  "kind": 1312,
+  "tags": [
+    ["a", "30311:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>", "from"],
+    ["a", "30311:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>", "to"],
+  ],
+  "content": "Zap this epic streamer!",
+  // other fields...
+}
+```
+
+### Stream Clips
+
+Event `kind:1313` is a live stream clip.
+
+A clip is a small video clip taken from the live stream used to highlight a specific memorable or entertaining part of the stream.
+
+Clients MUST include one `a` tag which points to the live stream event which the clip was taken from.
+
+Clients MUST include one `r` tag which is a URL pointing to the video clip file, preferably in MP4 format for maximum compatibility.
+
+Clients MAY also include a message about the clip in the `title` tag.
+
+```jsonc
+{
+  "kind": 1313,
+  "tags": [
+    ["a", "30311:<Community event author pubkey>:<d-identifier of the community>", "<Optional relay url>", "root"],
+    ["r", "https://example.com/my-clip.mp4"],
+    ["title", "Streamer gets rekt by bot!"]
+  ],
+  "content": "",
+  // other fields...
+}
+```
+
+### Stream Timeout
+
+Event `kind:1314` is a live chat moderation event that temporarily mutes a user from chatting.
```

Comment:
This seems like the right approach for moderation instead of the host having to update their k:10000 mute list

Is there any merit in having an optional `a` tag that restricts the timeout to a specific stream?

This also could make it possible for the host to designate moderator pubkeys for their streams, but that it out of scope of this PR

Reply:
> Is there any merit in having an optional a tag that restricts the timeout to a specific stream?

Maybe we could add that, but i think in 90% of cases you would just timeout based only on time, so if somebody is really rude you would do a timeout for 1 week for example and you would want to timeout them regardless of which stream, then in other cases a 30min timeout, the streamer knows how long their own stream is and can do the timeout without tagging the stream.

> This also could make it possible for the host to designate moderator pubkeys for their streams, but that it out of scope of this PR 

yea i want to add this in another PR, to create moderators pubkey set